source/auth/auth.rst

```diff
@@ -1180,65 +1180,15 @@ MONGODB-OIDC
 
 :since: 7.0 Enterprise
 
-MONGODB-OIDC authenticates using an `OIDC <https://openid.net/specs/openid-connect-core-1_0.html>`_ access tokens.  Drivers MUST support
-both Callback-driven OIDC and Automatic OIDC Authentication for AWS.
-
-
-Conversation
-````````````
-
-Authenticating using the MONGODB-OIDC mechanism will require 1 or 2 round trips between the MongoDB driver and server.  The requests from the driver and the replies from the server are described by the following IDL structs which are encoded in the payload as octet sequences defining BSON objects:
-
-.. code:: idl
-
-  PrincipalStepRequest:
-    description: Driver’s opening request in saslStart
-    fields:
-      n:
-        description: "Name of the OIDC user Principal"
-        type: string
-        optional: true
-
-Note that the principal name is optional as it may be provided by the IDP in environments where only one IDP is used.  The username provided by the user MUST be used as the principalName.
-
-.. code:: idl
-
-  IdPServerInfo:
-    description: "The information used by callbacks to authenticate with the Identity Provider."
-    fields:
-      issuer:
-        description: >-
-            URL which describes the Authentication Server. This identifier should be
-            the iss of provided access tokens, and be viable for RFC8414
-            metadata discovery and RFC9207 identification.
-        type:string
-      clientId:
-        description: "Unique client ID for this OIDC client"
-        type: string
-      requestScopes:
-        description: "Additional scopes to request from IDP"
-        type: array<string>
-        optional: true
-
-Server will use principalName (n) if provided in the driver’s PrincipalStepRequest to select an appropriate IDP.  This IDP's configuration will be returned in the server’s response that will be used by the end-user to acquire an Access Token.
-
-This Access Token will be used as the JWT in the driver’s JwtStepRequest to complete authentication.
-
-.. code:: idl
-
-  JwtStepRequest:
-      description: "Client's request with signed token"
-      fields:
-          jwt:
-              description: "Compact serialized JWT with signature"
-              cpp_name: JWT
-              type: string
+MONGODB-OIDC authenticates using an `OIDC <https://openid.net/specs/openid-connect-core-1_0.html>`_ access tokens.
+Drivers MAY implement the human authentication workflow with ``REQUEST_TOKEN_CALLBACK``.
 
 `MongoCredential`_ Properties
 `````````````````````````````
 
 username
-    MUST NOT be specified in automatic authentication. Drivers MUST allow the user to specify this in the callback-driven authentication. If a user omits this when multiple OIDC providers are configured, the server will produce an error during authentication.
+    MUST NOT be specified in machine authentication. Drivers MUST allow the user to specify this in the human authentication.
```

Comment:
Should this say
```
... in the human authentication workflow.
```
or 
```
... in the ``REQUEST_TOKEN_CALLBACK``.
```
?

Reply:
`in the human authentication workflow.`, updated.